packages/smooth_app/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: smooth_app
 description: A new official Open Food Facts application build with great user experience in mind.
-version: 0.0.0
+version: 0.0.0+600
```

Comment:
Is that a fact?

Reply:
The +600 allows to generate a versionCode.
On the Android app, the current is around 550